lms/resources/_js_config/file_picker_config.py

```diff
@@ -4,68 +4,67 @@ class FilePickerConfig:
     @classmethod
     def blackboard_config(cls, context, request, application_instance):
         """Get Blackboard files config."""
-        if not application_instance.product == application_instance.Product.BLACKBOARD:
-            return {}
-
-        enabled = application_instance.settings.get("blackboard", "files_enabled")
+        files_enabled = application_instance.settings.get("blackboard", "files_enabled")
+        groups_enabled = context.blackboard_groups_enabled
 
         auth_url = request.route_url("blackboard_api.oauth.authorize")
         course_id = request.params.get("context_id")
 
-        return {
-            "enabled": enabled,
-            "groupsEnabled": context.blackboard_groups_enabled,
-            "listFiles": {
+        config = {
+            "enabled": files_enabled,
+            "groupsEnabled": groups_enabled,
+        }
+
+        if files_enabled:
+            config["listFiles"] = {
                 "authUrl": auth_url,
                 "path": request.route_path(
                     "blackboard_api.courses.files.list", course_id=course_id
                 ),
-            },
-            "listGroupSets": {
+            }
+
+        if groups_enabled:
+            config["listGroupSets"] = {
                 "authUrl": auth_url,
                 "path": request.route_path(
                     "blackboard_api.courses.group_sets.list", course_id=course_id
                 ),
-            },
-        }
+            }
+
+        return config
 
     @classmethod
     def canvas_config(cls, context, request, application_instance):
         """Get Canvas files config."""
-
-        if not context.is_canvas:
-            return {}
-
         enabled = context.is_canvas and (
```

Comment:
This is a strange mix of `files_enabled`, `is_canvas` and `has_consumer_key`. I left it with the original name and don't use it to configure the config. I think we should add a flag like `files_enabled` so this behaves like other features. For another PR IMO.